shared/src/fixtures/arrow.rs

```diff
@@ -235,11 +239,11 @@ pub fn primitive_create_table(server: &str, table: &str) -> String {
             uint64_col        numeric(20),
             float32_col       real,
             float64_col       double precision,
-            timestamp_col     timestamp,
+            -- timestamp_col     bigint,
```

Comment:
Explain here as well why they're commented out

Reply:
I put up an issue https://github.com/paradedb/paradedb/issues/1299 for it here